# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmValue.h
# Commit: 1a73cfde72e8
# Full Hash: 1a73cfde72e8f4d19bdcfdbffa9fbc78bba01e24
# Author: Dmitry Bezhetskov <dbezhetskov@igalia.com>
# Date: 2021-09-28 09:54:33
# Regressor Bug: 1639153
# File Overlap Count: 1
# Description:
#   Bug 1639153 - Introduce indirect stubs to optimize call_indirect. r=lth
#   
#   This patch introduces indirect stubs.
#   An indirect stub is a stub that takes care of any switching activities needed for call_indirect.
#   Before this patch, we have to conservatively assume that any call_indirect's target can be from a foreign instance,
# ==============================================================================

diff -r da3c5d8492db -r 1a73cfde72e8 js/src/wasm/WasmValue.h
--- a/js/src/wasm/WasmValue.h	Tue Sep 28 05:16:23 2021 +0000
+++ b/js/src/wasm/WasmValue.h	Tue Sep 28 05:24:15 2021 +0000
@@ -239,7 +239,7 @@
 
   JSFunction* asJSFunction() { return value_; }
 
-  bool isNull() { return value_ == nullptr; }
+  bool isNull() const { return value_ == nullptr; }
 
   void trace(JSTracer* trc) const;
 };
